assets/javascript/game.js

```diff
@@ -0,0 +1,67 @@
+// GAME VARIABLES AND ARRAYS
+
+// An array of the alphabet used for computer's choices
+var computerChoices = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"];
+
+// Variables of 'wins', 'losses', 'guesses left' and 'current guesses' for tracking scores and guesses
+
+var wins = 0;
+var losses = 0;
+var guessesLeft = 10;
+
+// Variables to hold references and display them in HTML
+var userChoiceText = document.getElementById("userchoice-text");
+var winsText = document.getElementById("wins-text");
+var lossesText = document.getElementById("losses-text");
+var guessesLeftText = document.getElementById("guessesleft-text");
+
+// GAME FUNCTION 
+
+// Function that runs when user presses a key
+document.onkeyup = function (event) {
+    // Function runs only if user picks a letter from A-Z to limit other keys being pressed
+
+    // This determines when a key was pressed
+    var userGuess = event.key;
+
```

Comment:
You should try to look up on how to only get alphabet keys because currently itll take in any key that you choose